pkgs/by-name/pd/pdisk/package.nix

```diff
@@ -41,30 +39,30 @@ stdenv.mkDerivation rec {
       url = "https://aur.archlinux.org/cgit/aur.git/plain/linux_strerror.patch?h=pdisk&id=d0c930ea8bcac008bbd0ade1811133a625caea54";
       sha256 = "sha256-HGJIS+vTn6456KtaETutIgTPPBm2C9OHf1anG8yaJPo=";
     })
+
+    # Fix missing includes on Linux
+    ./cmdline.patch
```

Comment:
@OPNA2608 any chance to upstream ?

Reply:
I don't know where I'd have to send that since Apple doesn't seem to watch the PR sections of their repos, and I'm p sure they only maintain them for license compliance reasons or smth in the first place.

Reply:
fair enough